docs/release_notes/release_notes.md

```diff
@@ -12,6 +12,28 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
 {% endif %}
 
+## Notable Features
+
+### Puppet 4 Support
+
+Puppet 4 is now supported by Katello.
```

Comment:
@stbenjam I'll be sure to add that 
@ehelms @zjhuntin @cfouant @komidore64 anything else that we can say to users here?